lambda_code/logs/ResourcePolicy/index.py

```diff
@@ -0,0 +1,33 @@
+"""
+Custom resource to support Amazon Cloudwatch Logs ResourcePolicy.
+https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_ResourcePolicy.html
+"""
+
+from cfn_custom_resource import CloudFormationCustomResource
+
+
+class ResourcePolicy(CloudFormationCustomResource):
+
+    def validate(self):
+        self.policy_name = self.resource_properties['PolicyName']
+        self.policy_doc = self.resource_properties['PolicyDocument']
+
+    def create(self):
+        cl = self.get_boto3_client('logs')
+        cl.put_resource_policy(policyName=self.policy_name, policyDocument=self.policy_doc)
+        return {}
+
+    def update(self):
+        return self.create()
```

Comment:
Als je PolicyName wijzigt in een update, ga je op deze manier de oude policy laten staan.
Ik denk dat je beter de "PhysicalResourceId" zelf in handen neemt, en bijvoorbeeld de PolicyName ervoor gebruikt. [1]

[1]: https://github.com/iRobotCorporation/cfn-custom-resource/wiki/Basic-usage